src/thinkspace/api/thinkspace-readiness-assurance/app/models/thinkspace/readiness_assurance/assessment.rb

```diff
@@ -1,6 +1,10 @@
 module Thinkspace
   module ReadinessAssurance
     class Assessment < ActiveRecord::Base
+      ## Can get the after_save working via callbacks (http://stackoverflow.com/questions/1342761/how-to-skip-activerecord-callbacks?noredirect=1&lq=1)
```

Comment:
Look at: http://edgeguides.rubyonrails.org/active_record_callbacks.html#skipping-callbacks - it looks like we can use `update_column` to avoid the `after_save`.  There's a real use case where we end up modifying the JSON directly through a `rails c`.  Forgetting to update the tRAT would be easy.  I think we should try and hook up the `after_save` for now.